features/step_definitions/cluster_configuration/localstorage.rb

```diff
@@ -1,245 +1,95 @@
-# OCP supports fileystem local storage provisioner from 3.7
-# For some OCP versions, before depolying local storage provisoner,
-# we need enable some feature gates first.
-# ref: doc/storage_automation_spec.adoc
-
-Given /^I deploy local storage provisioner(?: with "([^ ]+?)" version)?$/ do |img_version|
+Given /^local storage operator has been installed successfully$/ do
   ensure_admin_tagged
-  ensure_destructive_tagged
-
-  namespace = env.local_storage_provisioner_project.name
-  config_hash = env.master_services[0].config.as_hash()
-  imgformat = config_hash["imageConfig"]["format"]
-  img_registry = imgformat.split("\/")[0]
-  ose_version = env.get_version(user: admin)
-  img_version = "v#{ose_version[0]}" unless img_version
-  serviceaccount="local-storage-admin"
-  configmap="local-volume-config"
-  template="local-storage-provisioner"
-  image="#{img_registry}/openshift3/local-storage-provisioner:#{img_version}"
-  path ||="/mnt/local-storage"
-  cmurl = "#{BushSlicer::HOME}/testdata/storage/localvolume/configmap-37.yaml"
-  if env.version_ge("3.10", user: user)
-    cmurl = "#{BushSlicer::HOME}/testdata/storage/localvolume/configmap.yaml"
-  end
-
-  project(namespace)
-  # if project exists, delete project and pvs created by local storage provisioner
-  if project.exists?(user: admin)
-    project.delete(by: admin)
-    project.wait_to_disappear(admin)
-
-    BushSlicer::PersistentVolume.list(user: admin).each { |pv|
-      pv.delete(by: admin) if pv.name.start_with?("local-pv-") && (pv.local_path&.start_with?("#{path}/fast") || pv.local_path&.start_with?("#{path}/slow"))
-    }
+  namespace = "openshift-local-storage"
+  name = "local-storage"
+
+  # create namespace
+  unless project(namespace).exists?
+    step %Q/I run the :oadm_new_project admin command with:/, table(%{
+      | project_name  | #{namespace} |
+    })
+    step %Q/the step should succeed/
   end
 
-  ns_cmd = "oc adm new-project #{namespace} --node-selector=''"
-  @result = env.master_hosts.first.exec(ns_cmd)
-  step %Q/the step should succeed/
-
-  env.hosts.each do |host|
-    setup_commands = [
-      "mkdir -p #{path}/fast/vol1",
-      "mount -t tmpfs fvol1 #{path}/fast/vol1",
-      "mkdir -p #{path}/fast/vol2",
-      "mount -t tmpfs fvol2 #{path}/fast/vol2",
-      "mkdir -p #{path}/slow/vol1",
-      "mount -t tmpfs svol1 #{path}/slow/vol1",
-      "mkdir -p #{path}/slow/vol2",
-      "mount -t tmpfs svol2 #{path}/slow/vol2",
-      "chcon -R unconfined_u:object_r:svirt_sandbox_file_t:s0 #{path}"
-    ]
-    res = host.exec_admin(*setup_commands)
-    raise "error preaparing subdirs for local storage provisioner" unless res[:success]
+  # create operator group
+  step %Q/I use the "#{namespace}" project/
+  unless operator_group(name).exists?
+    step %Q/I obtain test data file "storage\/localvolume\/operatorgroup.yaml"/
```

Comment:
@qinpingli to bypass escape, you can use something like `step %Q|....|` instead.  You are not limited to use `/` as a step delimiter.